Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift

```diff
@@ -33,14 +33,6 @@ public extension RichTextViewComponent {
         _ alignment: RichTextAlignment
     ) {
         if richTextAlignment == alignment { return }
-        if !hasTrimmedText {
-            let style = NSMutableParagraphStyle()
-            style.alignment = alignment.nativeAlignment
-            var attributes = richTextAttributes
-            attributes[.paragraphStyle] = style
-            typingAttributes = attributes
-        } else {
-            setRichTextAlignment(alignment, at: selectedRange)
-        }
+        setRichTextAlignment(alignment, at: selectedRange)
```

Comment:
I think the issue in here is that I am testing on iOS and you on macOS. :D 

This will work on macOS but not on iOS because it doesnt set the typing attributes and the behaviour of text is quite different. What I suggest is to platform-check this and use set typing attributes on non-mac platform, that way we can actually ensure it works properly and test those under both platforms.

Reply:
I actually tested continuously in  both iOS and macOS. :)

Reply:
Currently not setting typing attributes causes obviously this: 
https://github.com/danielsaidi/RichTextKit/assets/17381941/00fa4f8f-ce12-4992-9627-e09151c0e96c

I am on it, hopefully I can fix this today so lets see.